internal/cache/util/util.go

```diff
@@ -42,12 +43,14 @@ var (
 )
 
 const (
-	MiB              = 1024 * 1024
-	KiB              = 1024
-	DefaultFilePerm  = os.FileMode(0600)
-	DefaultDirPerm   = os.FileMode(0700)
-	FileCache        = "gcsfuse-file-cache"
-	BufferSizeForCRC = 65536
+	MiB                    = 1024 * 1024
+	KiB                    = 1024
+	DefaultFilePerm        = os.FileMode(0600)
+	DefaultDirPerm         = os.FileMode(0700)
+	FileCache              = "gcsfuse-file-cache"
+	BufferSizeForCRC       = 65536
+	SequentialReadSizeInMb = 22
+	CacheMaxSize           = 2 * SequentialReadSizeInMb * util.MiB
```

Comment:
same thing. test related stuff should be test files. Otherwise it creates confusion.